swerve/swerveModule.py

```diff
@@ -136,6 +135,9 @@ def __init__(self, location : tuple[float, float, str], channelBase: int, invert
         self.translation = wpimath.geometry.Translation2d(location[0], location[1])
         self.distTraveled = 0
 
+        self.steerAngle = 0.0
```

Comment:
Should this be in this changeset or are the changes to swerveModule from another branch? If the latter, we can help get this cleaned up so this is ONLY for telemetry.